fxfs/core/pfs/test/data/colors/orange

```diff
@@ -0,0 +1 @@
+orange
```

Comment:
Are these for testing? If so consider putting them in `testdata`. Lookup `testdata` convention in go.

Reply:
done